corehq/apps/domain/decorators.py

```diff
@@ -241,6 +241,22 @@ def __inner__(request, domain, *args, **new_kwargs):
     return decorator
 
 
+def api_domain_view(view):
+    """
+    Decorate this with any domain view that should be accessed via api only
+    """
+    @wraps(view)
+    @api_key()
+    @login_and_domain_required
+    def _inner(request, domain, *args, **kwargs):
+        if request.user.is_authenticated():
+            request.couch_user = CouchUser.from_django_user(request.user)
```

Comment:
looks like the above [two](https://github.com/dimagi/commcare-hq/blob/34c5f6dbfcdf8f06272263edbfe76b349c8963f8/corehq/apps/domain/decorators.py#L74) [lines](https://github.com/dimagi/commcare-hq/blob/34c5f6dbfcdf8f06272263edbfe76b349c8963f8/corehq/apps/domain/decorators.py#L82-L86) are redundant with `login_and_domain_required`